hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -450,7 +451,7 @@ private class DeleteIterator extends OzoneListingIterator {
         throws IOException {
       super(f);
       this.recursive = recursive;
-      if (getStatus().isDirectory()
+      if (getStatus().isDir()
```

Comment:
Isn't this a deprecated API and which is the same as isDirectory()?

Reply:
Yes! it's because Ozone uses a class called FileStatusAdapter, which is similar with the class FileStatus in hdfs. The main difference is that FileStatusAdapter class is built in with attribute of disk usage of file, which we'll use in the 'du' command!